src/crypto/store/base.js

```diff
@@ -9,3 +9,23 @@
  *
  * @interface CryptoStore
  */
+
+/**
+ * Represents an outgoing room key request
+ *
+ * @typedef {Object} OutgoingRoomKeyRequest
+ *
+ * @property {string} requestId    unique id for this request
```

Comment:
In absence of this being in the spec, please add a link to the gdoc so readers know where to go to find the wire protocol.